_platforms/it/io.md

```diff
@@ -16,17 +16,17 @@ github_team: io
 ## Intro
 
 io è l’applicazione del Governo italiano che permetterà a tutti i cittadini di usare servizi pubblici nazionali e locali dal proprio smartphone in maniera semplice, moderna e sicura.
-È un progetto Open Source che crescerà di giorno in giorno, coinvolgendo enti e servizi diversi.
+È un progetto Open Source che cresce di giorno in giorno, coinvolgendo enti e servizi diversi.
 
-io mira ad agevolare la relazione tra i cittadini e la Pubblica Amministrazione (PA) attraverso la creazione di una piattaforma di componenti riutilizzabili in grado di rendere i servizi digitali più efficaci e migliorare la comunicazione tra i cittadini e la PA.
+io mira ad agevolare la relazione tra i cittadini e la Pubblica Amministrazione (PA) attraverso la creazione di una piattaforma di componenti riutilizzabili in grado di rendere i servizi digitali più efficaci.
```

Comment:
It was sounding bad in Italian. I think that a better relationship means a better communication without having to say it again in the same sentence